test/cases/with-description-newlines.ts

```diff
@@ -0,0 +1,30 @@
+export var schema = {
+  "title": "Example Schema",
+  "description": "My cool schema",
+  "type": "object",
+  "properties": {
+    "firstName": {
+      "description": "first name single line description",
+      "type": "string"
+    },
+    "lastName": {
+      "type": "string"
+    },
+    "age": {
+      "description": "Age description with\nmultiple lines",
+      "type": "integer",
+      "minimum": 0
+    }
+  },
+  "required": ["firstName", "lastName"]
+}
+
+export var types = `/** My cool schema */
+export interface ExampleSchema {
+  firstName: string; // first name single line description
+  lastName: string;
+  // Age description with
+  // multiple lines
```

Comment:
Done in https://github.com/bcherny/json-schema-to-typescript/pull/26/commits/adace03e6f19311050885f91185ab39ad5caab45